steward/src/commands/schedule_corks/aave_cellar/fees_distributor.rs

```diff
@@ -0,0 +1,92 @@
+use crate::{
+    application::APP,
+    cellars::{self},
+    config,
+    prelude::*,
+    somm_send,
+};
+use abscissa_core::{clap::Parser, Command, Runnable};
+use deep_space::{Coin, Contact};
+use ethers::abi::AbiEncode;
+use somm_proto::cork::Cork;
+use std::time::Duration;
+use steward_abi::aave_v2_stablecoin::*;
+
+const MESSAGE_TIMEOUT: Duration = Duration::from_secs(10);
+const CHAIN_PREFIX: &str = "somm";
+
+/// Fees Distributor subcommand
+#[derive(Command, Debug, Parser)]
+pub struct FeesDistributorCmd {
+    #[clap(short = 'n', long)]
+    new_fees_distributor: String,
+
+    /// Target contract for scheduled cork.
+    #[clap(short = 'c', long)]
+    contract: String,
+
+    /// Block height to schedule cork.
+    #[clap(short = 'b', long)]
+    height: u64,
+}
+
+impl Runnable for FeesDistributorCmd {
+    fn run(&self) {
+        let config = APP.config();
+
+        abscissa_tokio::run_with_actix(&APP, async {
+            let mut bytes = [0u8; 32];
+            if let Err(err) = hex::decode_to_slice(self.new_fees_distributor.clone(), &mut bytes) {
```

Comment:
As discussed out of band, this will need to be formatted in the same way the orchestrator handles formatting Cosmos addresses.